### PR TITLE
Link table between event and req

### PR DESCRIPTION
**Pull request by anurag-ks**

### CODE REVIEW

**Comment by anurag-ks** [comment]
Please review the work.

File: modules/s3db/req.py [hunk 1]
```diff
@@ -1592,6 +1590,25 @@ def req_req_duplicate(item):
             item.id = duplicate.id
             item.method = item.METHOD.UPDATE
 
+
+# =============================================================================
+class S3RequestEventModel(S3Model):
```
Comment (hitesh96db) [inline modules/s3db/req.py]: I don't think there's a need to make a separate class for this. Could be placed in the same class.

File: modules/s3db/req.py [hunk 2]
```diff
@@ -564,6 +557,11 @@ def model(self):
                                                "joinby": "req_id",
                                                "key": "item_category_id",
                                                },
+                       # Request Event
```
Comment (hitesh96db) [inline modules/s3db/req.py:572]: Once this is a component, this should be added as an InlineComponent in the form as well. 
http://eden.sahanafoundation.org/wiki/S3/S3SQLForm shows how to add Components to forms.

**Comment by hitesh96db** [comment]
Please send a PR to the branch made specially for you.
hint: you might recognize the branch name ;)

**Pull request closed without merging**